README.md

```diff
@@ -1,58 +1,110 @@
+    - **Foundation Model:** Based on the insights from the 2024 CVPR paper [*Rethinking the Up-Sampling Operations in CNN-based Generative Network for Generalizable Deepfake Detection*](https://arxiv.org/abs/2312.10461), our primary model utilizes Neighborhood Pixel Relationships to detect specific discrepancies in AI-generated images.
+    - **Research Integration:** We systematically update our detection models and methodologies in response to emerging academic research, offering resources like training codes and model weights to our community.
+- **Validators:** Responsible for challenging miners with a balanced mix of real and synthetic images, drawn from a diverse pool of sources.
+    - **Resource Expansion:** We are committed to enhancing the validators' capabilities by increasing the diversity and volume of the image pool, which supports rigorous testing and validation processes.
 
-<center>
-    <img src="static/Subnet-Arch.png" alt="Subnet Architecture"/>
-</center>
+**Contribution:**
+Developers and researchers are invited to contribute to the BitMind Subnet, whether by participating in model development, contributing to codebases, or engaging with our community. Your involvement helps improve the efficacy of digital media verification technologies.
 
+**Stay Updated:**
+For upcoming features and community updates, please follow our project repository and join our discussions.
 
-## Status
+![Subnet Architecture](static/Subnet-Arch.png)
 
-We are on testnet, uid 168!
+We are currently in testnet phase, uid 168.
 
-[Join our Discord!](https://discord.gg/SaFbkGkU)
+## Join Our Discord Community
+
+<p align="left">
+  <a href="https://discord.gg/bitmind">
+    <img src="static/Join-BitMind-Discord.png" alt="Join us on Discord" width="60%">
+  </a>
+</p>
+
+For real-time discussions, community support, and regular updates, join our Discord server. Connect with developers, researchers, and users to get the most out of BitMind Subnet.
+
+## Project Structure and Terminology
+
+### Overview and Terminology
+
+Before diving into the specifics of the directory structure and key components, let's familiarize ourselves with the essential terms used throughout this project. Understanding these terms is crucial for navigating and contributing to the BitMind Subnet effectively. For a more detailed explanation of the terminology, please refer to [Bittensor Building Blocks](https://docs.bittensor.com/learn/bittensor-building-blocks).
+
+- **Synapse**: Acts as a communication bridge between axons (servers) and dendrites (clients), facilitating data flow and processing.
+- **Neuron**: A fundamental unit that includes both an axon and a dendrite, enabling full participation in the network operations.
+
+### Notable Directories
+
+- **bitmind/**: This directory contains the specific implementations of Bittensor operations, which include the key components such as miners, validators, and neurons.
```

Comment:
maybe a good idea to mention here that this code is used both by validators/miners as well as the base_miner training/eval code